src/multitask_personalization/rom/models.py

```diff
@@ -237,22 +243,24 @@ def set_trainable_parameters(self, params: Any) -> None:
         self._radius = params
 
     def check_position_reachable(
-        self, position: NDArray, padding: float = 1e-6
+        self, position: NDArray, padding: float = 1e-4
     ) -> bool:
         distance = float(np.linalg.norm(position - self._sphere_center))
-        return distance < self._radius + padding
+        reachable = distance < self._radius + padding
+        return reachable
```

Comment:
Curious why this change was made.... mainly interpretability?

Reply:
oops this is just left over from me printing out the value of `reachable` for debugging. I guess I'll leave it in case we want to do that again int he future